docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
                 - INSTALL_WORKSPACE_SSH=false
                 - PUID=1000
                 - PGID=1000
-                - NODE_VERSION=stable
+                - NODE_VERSION=6
```

Comment:
I tend to agree with you that it is far preferable in the long-term to lock the node version but for new users I think its better to have something that offers the latest version upon which they can build. Most LaraDock users end up modifying the `docker-compose` file anyhow.

We've discussed this previously in #368.